src/modules/chat_highlight_blacklist_keywords/index.js

```diff
@@ -42,266 +44,278 @@ const pinnedHighlightTemplate = ({timestamp, from, message}) => `
-    highlightUsers = computedUsers;
+  const keywords = defaultHighlightKeywords(storage.get('highlightKeywords'));
+  const {computedKeywords, computedUsers} = computeKeywords(keywords);
+  highlightKeywords = computedKeywords;
+  highlightUsers = computedUsers;
 }
 
 function wildcard(keyword) {
-    return keyword.replace(/\*/g, '[^ ]*');
+  return keyword.replace(/\*/g, '[^ ]*');
 }
 
 function exactMatch(keyword) {
-    return keyword.replace(/^<(.*)>$/g, '^$1$$');
+  return keyword.replace(/^<(.*)>$/g, '^$1$$');
 }
 
 function keywordRegEx(keyword) {
-    return new RegExp(`(\\s|^|@)${keyword}([!.,:';?/]|\\s|\$)`, 'i');
+  return new RegExp(`(\\s|^|@)${keyword}([!.,:';?/]|\\s|$)`, 'i');
 }
 
 function fromContainsKeyword(keywords, from) {
-    for (const user of keywords) {
-        if (user.toLowerCase() !== from) continue;
-        return true;
-    }
-    return false;
+  for (const user of keywords) {
+    if (user.toLowerCase() !== from) continue;
+    return true;
+  }
+  return false;
 }
 
 function messageContainsKeyword(keywords, from, message) {
-    for (let keyword of keywords) {
-        keyword = escapeRegExp(keyword);
-        keyword = wildcard(keyword);
-        keyword = exactMatch(keyword);
-
-        const currentUser = twitch.getCurrentUser();
-        const filterCurrentUser = (currentUser && from !== currentUser.name) || !currentUser;
-        if (filterCurrentUser && keywordRegEx(keyword).test(message)) return true;
-    }
-    return false;
+  for (let keyword of keywords) {
+    keyword = escapeRegExp(keyword);
+    keyword = wildcard(keyword);
+    keyword = exactMatch(keyword);
+
+    const currentUser = twitch.getCurrentUser();
+    const filterCurrentUser = (currentUser && from !== currentUser.name) || !currentUser;
+    if (filterCurrentUser && keywordRegEx(keyword).test(message)) return true;
+  }
+  return false;
 }
 
 function isReply($message) {
-    return $message.parent().hasClass('chat-input-tray__open');
+  return $message.parent().hasClass('chat-input-tray__open');
 }
 
 function messageTextFromAST(ast) {
-    return ast.map(node => {
-        switch (node.type) {
-            case 0: // Text
-                return node.content.trim();
-            case 3: // CurrentUserHighlight
-                return node.content;
-            case 4: // Mention
-                return node.content.recipient;
-            case 5: // Link
-                return node.content.url;
-            case 6: // Emote
-                return node.content.alt;
-        }
-    }).join(' ');
+  return ast
+    .map((node) => {
+      switch (node.type) {
+        case 0: // Text
+          return node.content.trim();
+        case 3: // CurrentUserHighlight
+          return node.content;
+        case 4: // Mention
+          return node.content.recipient;
+        case 5: // Link
+          return node.content.url;
+        case 6: // Emote
+          return node.content.alt;
+        default:
+          return false;
```

Comment:
why would this return false?

Reply:
.map expects a return, and returning false is essentially doing a continue

Reply:
empty string goes here